cypress/e2e/shared/dashboardsIndex.test.ts

```diff
@@ -761,4 +762,147 @@ describe('Dashboards', () => {
       })
     })
   })
+  it('changes time range', () => {
+    const dashName = 'dashboard'
+    cy.writeLPData({
+      lines: genCurve({points: 87000, freq: 10}),
+      offset: `${30 * 24 * 60}m`,
+      stagger: `${(30 * 24 * 60) / 87000}m`,
+    })
+    cy.intercept('POST', '/api/v2/query?*', req => {
+      req.alias = 'loadQuery'
+    })
+    cy.get<Organization>('@org').then(({id}: Organization) =>
+      cy.createDashboard(id, dashName).then(({body}) => {
```

Comment:
remove the curl brace here to since we're not explicitly declaring a `return` to handle the cypress async tasks correctly

Reply:
This is still an issue. Please address it. Thanks

Reply:
I removed the curl brace. Is that the issue you were reffering to?